source/_components/here_travel_time.markdown

```diff
@@ -0,0 +1,145 @@
+  required: true
+  type: string
+origin:
+  description: "The starting point for calculating travel distance and time."
+  required: true
+  type: string
+destination:
+  description: "One or more locations to use as the finishing point for calculating travel distance and time."
+  required: true
+  type: string
+name:
+  description: A name to display on the sensor. The default is "HERE Travel Time".
+  required: false
+  type: string
+travel_mode:
+  description: "You can choose between: `car`, `pedestrian`, `publicTransport` or `truck`. The default is `car`."
+  required: false
+  type: string
+route_mode:
+  description: "You can choose between: `fastest`, or `shortest`. This will determine whether the route is optimized to be the shortest and completely disregard traffic and speed limits or the fastest route according to the current traffic information. The default is `fastest`"
+  required: false
+  type: string
+traffic_mode:
+  description: "You can choose between: `true`, or `false`. Decide whether you want to take the current traffic condition into account. Default is `false`."
+  required: false
+  type: string
+{% endconfiguration %}
+
+## {% linkable_title Dynamic Configuration %}
+
+Tracking can be setup to track entities of type `device_tracker`, `zone`, `sensor` and `person`. If an entity is placed in the origin or destination then every 5 minutes when the platform updates it will use the latest location of that entity.
+
+```yaml
+# Example entry for configuration.yaml
+sensor:
+  # Tracking entity to entity
+  - platform: here_travel_time
+    app_id: "YOUR_APP_ID"
+    app_code: "YOUR_APP_CODE"
+    name: Phone To Home
+    origin: device_tracker.mobile_phone
+    destination: zone.home
+
+  # Tracking entity to zone friendly name
+  - platform: here_travel_time
+    app_id: "YOUR_APP_ID"
+    app_code: "YOUR_APP_CODE"
+    name: Home To Eddie's House
+    origin: zone.home
+    destination: Eddies House    # Friendly name of a zone
+```
+
+## {% linkable_title Entity Tracking %}
+
+- **device_tracker**
+  - If state is a zone then the zone location will be used
+  - If state is not a zone it will look for the longitude and latitude attributes
+- **zone**
+  - Uses the longitude and latitude attributes
+  - Can also be referenced by just the zone's friendly name found in the attributes.
+- **sensor**
+  - If state is a zone or zone friendly name then will use the zone location
+  - All other states will be passed directly into the HERE API
+    - This includes all valid locations listed in the *Configuration Variables*
+
+## {% linkable_title Updating sensors on-demand using Automation %}
+
+You can also use the `homeassistant.update_entity` service to update the sensor on-demand. For example, if you want to update `sensor.morning_commute` every 2 minutes on weekday mornings, you can use the following automation:
+
+```yaml
+- id: update_morning_commute_sensor
```

Comment:
You might want to start this with `automation:` to indicate more clearly that this is an automation.